browser/android/constants.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_ANDROID_CONSTANTS_H_
+#define BRAVE_BROWSER_ANDROID_CONSTANTS_H_
+
+namespace settings {
+
+// GENERATED_JAVA_ENUM_PACKAGE: org.chromium.chrome.browser.settings
+enum class YTVideoQuality { kOn, kAllowOverWifi, kOff };
```

Comment:
youtube_constants.h and also please don't abbreviate. Use Youtube/youtube instead of YT in general for clarity